components/src/components/BCAddress/Component.ts

```diff
@@ -16,80 +15,82 @@ const ID = 'bcaddress';
 const DISPLAY = 'BC Address';
 
 export default class Component extends (ParentComponent as any) {
-    static schema(...extend) {
-
-        return ParentComponent.schema({
-            label: DISPLAY,
-            type: ID,
-            key: ID,
-            provider: 'custom',
-            providerOptions: {
-                queryProperty: 'addressString',
-
-              url: import.meta.env.VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL},
-
-            queryParameters:{"echo": false,
-            "brief": true,
-            "minScore": 55,
-            "onlyCivic": true,
-            "maxResults": 15,
-            "autocomplete": true,
-            "matchAccuracy": 100,
-            "matchPrecision": "occupant, unit, site, civic_number, intersection, block, street, locality, province",
-            "precisionPoints": 100,}
-
-        }, ...extend);
-    }
-
-    static get builderInfo() {
-        return {
-            title: DISPLAY,
-            group: 'advanced',
-            icon: 'address-book',
-            weight: 90,
-            documentation: Constants.DEFAULT_HELP_LINK,
-            schema: Component.schema()
-        };
-    }
-
-    mergeSchema(component = {}) {
-
-      let components = component['components'];
-
-      if (components) {
-        return _.omit(component, 'components');
-      }
-
-      return component;
+  static schema(...extend) {
+    return ParentComponent.schema(
+      {
+        label: DISPLAY,
+        type: ID,
+        key: ID,
+        provider: 'custom',
+        providerOptions: {
+          queryProperty: 'addressString',
+          url: import.meta.env.VITE_CHEFS_GEO_ADDRESS_APIURL,
```

Comment:
This is the only code change in the file - use `VITE_CHEFS_GEO_ADDRESS_APIURL` instead of `VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL`.